chat-server/src/integrations/openai.ts

```diff
@@ -9,6 +9,15 @@ import {
 } from "@azure/openai";
 import { stripIndent } from "common-tags";
 
+export type OpenAiMessageEnum = "system" | "assistant" | "user";
```

Comment:
[nit] "Enum" is not super meaningful in this context
```suggestion
export type OpenAiMessageRole = "system" | "assistant" | "user";
```